packages/misskey-js/src/entities.ts

```diff
@@ -183,3 +183,29 @@ export type EmojiDeleted = {
 export type AnnouncementCreated = {
 	announcement: Announcement;
 };
+
+export type SignupRequest = {
+	username: string;
+	password: string;
+	host?: string;
+	invitationCode?: string;
+	emailAddress?: string;
+	'hcaptcha-response'?: string | null;
+	'g-recaptcha-response'?: string | null;
+	'turnstile-response'?: string | null;
+}
+
+export type SignupResponse = UserDetailed & {
```

Comment:
ここは `MeDetailed` のほうが良さそうです
```suggestion
export type SignupResponse = MeDetailed & {
```

Reply:
なおしてみました

https://github.com/misskey-dev/misskey/pull/12846/files#diff-737f5f4066ce5516ca2940bd0f6c2d40e5e214fa9caec59d0138cc144871c0a8R187-R220